### DISCUSSION TITLE
Why only Cloudinary?

### DISCUSSION THREAD

**Discussion by cloudcompute** [tinacms/tinacms#2882] [General]
Hi maintainers

Paying few dollars to Tina Cloud is okay because you you guys are spending your time to maintain this open source.

Cloudinary charges 29 dollars per month. For my business website which does not have more than 20 images, I am not going to pay 29 $ a month.

Is there any other alternative?

**Comment by perkinsjr** [comment]
We picked Cloudinary, in the beginning, to allow us to offer Media management as it's used widely. We did this to allow us to move quickly and finish more of the core product before our 1.0. 


> Cloudinary charges 29 dollars per month. For my business website which does not have more than 20 images, I am not going to pay 29 $ a month.

Cloudinary does have a free tier that should be more than ample for 20 images, I use the free tier to power all images throughout my blog (100s) https://cloudinary.com/pricing

We are exploring a DAM that is git-backed which has been a discussion point since 2021, this would allow an alternative without any additional 3rd party systems. We want to have this option for our 1.0 release.

**Reply by cloudcompute** [reply]
Thank you for the clarification. 

> Cloudinary does have a free tier that should be more than ample for 20 images.

I had the wrong impression about Cloudinary. If this is so, good enough (for the time being).

> We are exploring a DAM that is git-backed

I have not come across any react-based DAM so far. But yes, there is sanity.io's media plugin which is MIT-licensed and seems to be pretty stable. Here is its [link](
https://github.com/robinpyon/sanity-plugin-media) You may tweak it and replace Cloudinary.
